cg_hermes/config/raredisease.py

```diff
@@ -0,0 +1,147 @@
+"""Tags that are defined in Tomte deliverables mapped to tags used in CG
+
+The tag sets that exists in all files are set to mandatory. Tag sets that exists in < 4 deliverables are not
+mandatory by default. However, the tags that are available to a particular analysis is mandatory for that analysis.
+"""
+
+from cg_hermes.config.nextflow import NEXTFLOW_TAGS
+
+
+RAREDISEASE_COMMON_TAGS = {
```

Comment:
You might be missing:
- nextflow-samplesheet and nextflow-config (stored for future reproduciblity)
- multiqc intermediate files required by Janus

Reply:
Added